src/mixins/filing-template-mixin.ts

```diff
@@ -123,7 +123,12 @@ export default class FilingTemplateMixin extends DateMixin {
         date: this.getCurrentDate, // "absolute day" (YYYY-MM-DD in Pacific time)
         folioNumber: this.getFolioNumber // folio number, unless overridden below
       },
-      business: this.getEntitySnapshot.businessInfo,
+      business: {
+        foundingDate: this.getEntitySnapshot.businessInfo.foundingDate,
```

Comment:
If I'm not mistaken, we only need this in IAs and Registrations. After my investigation and with Argus' notes:
- business.legalType is needed everywhere
- business.founding date is needed in IAs and Registrations
- business.identifier is needed IAs and Registrations
- business.legalName is needed in Alterations

Argus' notes: https://community.inkdrop.app/note/34f49807029b14c784418ae85fa89695/note:0chMrbv87
I'll double check with Argus regarding that.

Reply:
Hmm, that's complicated. Is there is any problem with setting all 4 properties?